### ISSUE TITLE
Wrong rotating angle on pasted shapes on Flare

### ISSUE THREAD

**Issue by CLucera** [2d-inc/support#184]
There is a strange behaviour when pasting a shape and trying to rotate it

steps to reproduce:
- Create a rectangle shape
- select the new created shape
- copy and paste (ctrl+c - ctrl+v)
- select the new created shape and insert 45° as rotate

the shape will be rotated by 90 degree instead of 45, rotating it by 90 rotate it by 180 degree and so on

**Comment by neurowave** [comment]
Good catch! I see what's happening. When you paste a shape layer, the shape layer and the path layer below it both get selected. So if you apply a rotation with both of them selected, you are applying two rotations (one to the shape layer, and one to the path layer).

Suggested fix: when pasting a shape layer, only the new shape layer should be selected. Paths below it should be unselected. @luigi-rosso
